cmake/nanobind-config.cmake

```diff
@@ -421,6 +421,8 @@ function (nanobind_add_stub name)
     list(APPEND NB_STUBGEN_OUTPUTS "${ARG_OUTPUT}")
   endif()
 
+  file(TO_CMAKE_PATH ${Python_EXECUTABLE} Python_EXECUTABLE)
```

Comment:
Would it not be better to give this an internal name, e.g. `NB_PYTHON_EXECUTABLE`?

Reply:
Yeah, it is better. Will change it.